tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -51,3 +51,23 @@ Feature: full text search
       | dav-path-version |
       | old              |
       | new              |
+
+
+  Scenario Outline: search files using a deleted tag
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
+    And user "Alice" has created the following tags for file "file1.txt" of the space "Personal":
+      | tag1 |
+    When user "Alice" removes the following tags for file "file1.txt" of space "Personal":
+      | tag1 |
+    And user "Alice" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result should contain "0" entries
+    And the search result of user "Alice" should not contain these entries:
+      | file1.txt |
```

Comment:
would adding the step like so make sense in this case? To ensure that removing the tag won't remove the file
```
And file "file1.txt" should exists for user "Alice"
```

Reply:
I think we should use one of this two-step to verify that removing tag actually removed tag from files rather than both of them
```
And the search result should contain "0" entries
And the search result of user "Alice" should not contain these entries:
    | file1.txt |
```

Reply:
.....